### PR TITLE
wireguard: update docs for installing tools

### PR DESCRIPTION
**Pull request by riyazdf**
</img>

### CODE REVIEW

File: docs/wireguard.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ A full technical paper from NDSS 2017 is available [here](https://www.wireguard.
 The default kernels build WireGuard in as a module.
 
 ### Userspace Tools
-The userspace tools are now a package available in `tools/alpine`, which can be installed via `apk add wireguard-tools`.
+The userspace tools can be installed via `apk add -U -X http://dl-cdn.alpinelinux.org/alpine/edge/testing wireguard-tools` in the `getty` and `sshd` containers.
+
+The tools are also installed in the base `linuxkit/alpine` image for use when building system containers.
```
Comment (ijc) [inline docs/wireguard.md]: Don't know if it is worth mentioning here, given the previous paragraph, that a simple `apk add` is fine in this context.

It also depends on what happens to #2270 I suppose...

Comment (riyazdf) [inline docs/wireguard.md]: makes sense for now.  Added.

**Comment by zx2c4** [comment]
LGTM. 👍

**Comment by justincormack** [comment]
Can you just add this to #2270 it only makes sense if that revert is merged

**Comment by riyazdf** [comment]
@justincormack will do.  Closing this.

**Pull request closed without merging**